account_cost_center/migrations/13.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,13 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import SUPERUSER_ID, api
+
+
+def migrate(cr, version):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    rule = env.ref(
+        "account_cost_center.account_cost_center_comp_rule", raise_if_not_found=False,
+    )
+    if rule:
+        domain = "['|',('company_id','=',False),('company_id','in',company_ids)]"
+        rule.write({"domain_force": domain})
```

Comment:
I may be wrong here, I'm not very familiar with how OpenUpgrade works, so correct me if this isn't right. But my understanding is that since we had fields on invoices and invoice lines in v12 (Default Cost Center and Cost Center respectively), this module is responsible for migrating that data to `account.move` and `account.move.line` in v13, as `account.invoice` was merged into `account.move` and `account.invoice.line` was merged into `account.move.line`. They are different tables, so if you had data in one table in version 12, it will not automatically move over to the new table in version 13.

I'm not sure if there's some established way to do the migration of custom fields on invoices and invoice lines in OpenUpgrade: if you know it handles this automatically then we can just chalk this up to me not being able to google things properly :sweat_smile: 

If it doesn't handle it, then this migration script will need to be expanded to handle that.